lib/role_control/parental_controlled.rb

```diff
@@ -22,7 +22,13 @@ def parent_class
 
       def scope_for(action, user, opts={})
         parent_scope = parent_class.scope_for(action, user, opts)
-        joins(@parent).merge(parent_scope)
+
+        if Panoptes.flipper["no_join_parental_scope"].enabled?
```

Comment:
Agree, initially i was writing this as a code experiment but realised (again) that creating scopes doesn't actually execute the code, then i kept the naming of the conditional branch as it's not default behaviour...i.e. what i want to test. 

Flipper is great in that you don't have to make a feature for it to be tested against, it'll happily load something that doesn't exist, it'll just never be enabled until you do so via the UI.